python/lsst/ctrl/execute/libexec/dagIdInfo.py

```diff
@@ -26,9 +26,10 @@
 import re
 import sys
 
+
 # extracts a line from a DAG file to show which ids were processed for a
 # particular dag node
-if __name__ == "__main__":
+def main():
```

Comment:
I think you want this to return the exit status since I think the python scripts interface is going to call this as `sys.exit(main())`. This comment applies to all the `main()` functions here.